rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -6988,6 +7032,18 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 
 	void SUMB(spu_opcode_t op)
 	{
+		// TODO: Some future CPUS will support VNNI but not avx512
```

Comment:
What do you mean future CPUs? Also this already needs a separate feature detection because VNNI is supported on Cascade lake, Cooper lake.

Reply:
Alder Lake will not support AVX-512 on desktop, but will include VNNI.
https://en.wikipedia.org/wiki/Advanced_Vector_Extensions#AVX-VNNI

Reply:
Added handling for VNNI cpus. Alderlake and Sapphirerapids cpus won't be detected until the LLVM fork is updated though.